log/logrus/logrus_logger.go

```diff
@@ -11,13 +11,13 @@ import (
 )
 
 type logrusLogger struct {
-	*logrus.Logger
+	logrus.FieldLogger
 }
 
 var errMissingValue = errors.New("(MISSING)")
 
-// NewLogrusLogger returns a go-kit log.Logger that sends log events to a Logrus logger.
-func NewLogrusLogger(logger *logrus.Logger) log.Logger {
+// NewLogrusLogger returns a go-kit log.FieldLogger that sends log events to a Logrus logger.
```

Comment:
The function still returns a go-kit log.Logger. It's the input type that has changed to accept the logrus.FieldLogger type.

The phrase "... to a Logrus logger" still seems accurate and perhaps more to the point. Ending with "... to a logrus.FieldLogger" would be more precise, but just repeating what the code says. I'm not a Logrus user, so I'm not sure which phrasing is more intuitive for that user base. Thoughts?

```suggestion
// NewLogrusLogger returns a go-kit log.Logger that sends log events to a Logrus logger.
```

Reply:
Ah, good catch, it was kinda automatic change. 🙂

Reply:
![image](https://user-images.githubusercontent.com/12751435/49821861-07b15880-fdae-11e8-8153-1ec22afbd6eb.png)
From Github suggest great platform. 👍